src/conf_mode/protocols_bgp.py

```diff
@@ -301,6 +302,17 @@ def verify(bgp):
                     raise ConfigError(f'Specified peer-group "{peer_group}" for '\
                                       f'neighbor "{neighbor}" does not exist!')
 
+                if 'remote_as' in peer_config:
+                    is_ibgp = True
+                    if peer_config['remote_as'] != 'internal' and \
+                            peer_config['remote_as'] != bgp['system_as']:
+                        is_ibgp = False
+                    if peer_groups_context.get(peer_group) is None:
+                        peer_groups_context[peer_group] = is_ibgp
+                    elif peer_groups_context.get(peer_group) != is_ibgp:
```

Comment:
Please us: `elif peer_groups_context[peer_group] != is_ibgp:` to keep a consistent style accross this complex file.

Reply:
done